### PR TITLE
Make update-copyright end files with a newline

### LINKED ISSUE #48: Make update-copyright end with empty lines
Without this the `.rc` files do not compile.

### PR DESCRIPTION
**Pull request by folmos-at-orange**

### CODE REVIEW

**Comment by folmos-at-orange** [comment]
From the issue #48 @marcboulle 
> The CMakeLists.txt files should be updated by adding the related .rc at the end of the add_library list, in order to activate the Windows RC compiler, for the following projects: MODL, MODL_Coclustering, KhiopsNativeInterface.

Ok. I'm fixing it.

**Comment by folmos-at-orange** [comment]
@marcboulle 

I added the `.rc` files to the config and it compiles.

File: src/Learning/MODL/CMakeLists.txt [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-                    MDKhiopsModelingSpec.cpp MDKhiopsModelingSpecView.cpp MODL.cpp)
+add_executable(MODL${MPI_SUFFIX} MDKhiopsLearningProblem.cpp MDKhiopsLearningProblemView.cpp
+                                 MDKhiopsLearningProject.cpp MODL.cpp MODL.rc)
 
 target_link_libraries(MODL${MPI_SUFFIX} PUBLIC DTForest KMDRRuleLibrary KWLearningProblem PLParallelTask)
 set_khiops_options(MODL${MPI_SUFFIX})
 
 add_library(MODL_SO SHARED MDKhiopsLearningProblem.cpp MDKhiopsLearningProblemView.cpp MDKhiopsLearningProject.cpp
-                           MDKhiopsModelingSpec.cpp MDKhiopsModelingSpecView.cpp MODL_dll.cpp)
+                           MODL_dll.cpp)
```
Comment (marcboulle) [inline src/Learning/MODL/CMakeLists.txt]: Revert to V10 version

File: src/Learning/MODL/CMakeLists.txt [hunk 2]
```diff
@@ -1,13 +1,12 @@
 # on fedora, binaries built with mpi must ended by _mpich suffix
-add_executable(
-  MODL${MPI_SUFFIX} MDKhiopsLearningProblem.cpp MDKhiopsLearningProblemView.cpp MDKhiopsLearningProject.cpp
-                    MDKhiopsModelingSpec.cpp MDKhiopsModelingSpecView.cpp MODL.cpp)
+add_executable(MODL${MPI_SUFFIX} MDKhiopsLearningProblem.cpp MDKhiopsLearningProblemView.cpp
+                                 MDKhiopsLearningProject.cpp MODL.cpp MODL.rc)
```
Comment (marcboulle) [inline src/Learning/MODL/CMakeLists.txt]: Revert to V10 version, but keep the rc file

**Review by marcboulle — CHANGES_REQUESTED** [review]

File: src/Learning/MODL/CMakeLists.txt (hunk 2 above)
Comment (folmos-at-orange) [reply src/Learning/MODL/CMakeLists.txt]: Done

File: src/Learning/MODL/CMakeLists.txt (hunk 1 above)
Comment (folmos-at-orange) [reply src/Learning/MODL/CMakeLists.txt]: Done

**Review by marcboulle — APPROVED** [review]

**Pull request merged**